frontend/src/Resources/index.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  View, Image, Text,
+  StyleSheet,
+} from 'react-native';
+import factoryIcon from '../../assets/buildings/factory.png';
+import mineIcon from '../../assets/buildings/mine.png';
+import cookieIcon from '../../assets/troop/cookie.png';
+import goldIcon from '../../assets/gold.png';
+import Colors from '../common/colors';
+import { CardView } from '../common/components';
+
+const styles = StyleSheet.create({
+  text: {
+    fontSize: 16,
+    color: Colors.textColor,
+    fontWeight: 'bold',
+  },
+  subText: {
+    fontSize: 12,
+    color: Colors.greenColor,
+  },
+  resourceIcon: {
+    width: 14,
+    height: 14,
+    marginHorizontal: 4,
+  },
+  buildingIcon: {
+    width: 32,
+    height: 32,
+    marginHorizontal: 4,
+  },
+  rowFlex: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  resourceView: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginRight: 12,
+  },
+});
+
+function ResourceView({
+  textColor, buildingIcon,
+  resourceIcon, amount, changeRate,
+}) {
+  let changeRateStr;
+  if (changeRate < 0) {
+    changeRateStr = `${changeRate}/minute`;
+  } else {
+    changeRateStr = `+${changeRate}/minute`;
+  }
+
+  return (
```

Comment:
what's happening if there is an error during the fetching, so the `FETCH_RESOURCES_FAILURE` branch will be executed?

Reply:
I will add error handle when connecting the back end. It should navigate to the error page.